.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -293,6 +293,20 @@ jobs:
           follow-up-finalization-check: true
         timeout-minutes: 10
 
+  run-e2e-change-non-reserved:
+    needs: [build-test-docker, build-test-client]
+    name: Run non-reserved change test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Run e2e test
+        uses: ./.github/actions/run-e2e-test
+        with:
+          test-case: change_non_reserved
+          follow-up-finalization-check: true
+        timeout-minutes: 15
```

Comment:
10 should be sufficient here

Reply:
Add the check in the same test? Then I should probably also change name of the test, because it won't really reflect what it's doing?